### PR TITLE
costa: import pip cache and pyIndex to pip process

### PR DESCRIPTION
**Pull request by FeelyChau**
Fix: #576 .

### CODE REVIEW

File: packages/costa/src/runtime.ts [hunk 1]
```diff
@@ -363,7 +363,12 @@ export class CostaRuntime {
         '--default-timeout=1000',
         `--cache-dir=${this.options.installDir}/.pip`
       ]);
-      return spawnAsync(`${envDir}/bin/pip3`, args, {}, stdio);
+      return spawnAsync(`${envDir}/bin/pip3`, args, {
```
Comment (yorkie) [inline packages/costa/src/runtime.ts:366]: Using [`--install-option`](https://pip.pypa.io/en/stable/reference/pip_install/#install-install-option) might be a standard way.

**Comment by yorkie** [comment]
BTW, I'm still looking for a standard way to address #576 in the Python ecosystem.

**Comment by WenheLI** [comment]
This could fix detectron.

**Comment by yorkie** [comment]
What's the progress of this pull request?

**Comment by FeelyChau** [comment]
#574 has been fixed by #586 .

**Pull request closed without merging**